tests/thread_safe_logger/run_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+pushd ../.. > /dev/null 2>&1
+
+output="$(./build/bin/thread_safe_logger -thread_count 4 -run_time_ms 1000 --logall 2>&1)";
```

Comment:
1 seconds should be more than enough to start up 4 threads, even though the first threads probably will be able to output a few lines before the others start. I can test to ensure this isn't an issue at all.